tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+
+If you prefer to develop in the cloud without installing anything on your computer, SAP offers a great cloud-based development experience alternative in the form of the SAP Business Application Studio. Follow [this tutorial](appstudio-onboarding) for more details on getting your own instance. The rest of this tutorial assumes that you installed Node.js and npm on your computer.
+
+[DONE]
+
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Prepare package.json)]
+
+Select a folder (directory) where you want to develop your application. In this folder, you will create a small file called `package.json`, with the following content:
+
+```
+{
+    "name": "hello-graph",
+    "version": "0.0.0",
+    "private": true,
+    "dependencies": {
+        "express": "4.17.1",
+        "node-fetch": "2.6.1",
+        "universal-cookie": "4.0.4"
+    }
+}
+```
+After saving the file, run the following command on your console:
+
+```
+ npm install
+```
+If your node.js environment was properly set up, then this command will install a few standard library packages in a new sub-folder called `node_modules`.
+
+Now, create another sub-folder, next to `node_modules`, called `src`. This is where we will develop our code.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Prepare graph.js)]
+
+First, we will create a boilerplate file in the `src` folder, called `graph.js`. This file will contain a class called `Graph`, which provides a nice wrapper for using SAP Graph. At this time, we will show how to read data, via a `get()` function.
+
+Paste the following code into the file and save it. As you can see, it is very simple, and takes advantage of the node-fetch package.
+
+```
+const fetch = require("node-fetch");
+class Graph {
+    async get(req, entity, params) {
+        const url = `https://sandbox.api.sap.com/sapgraph/${entity}${params ? `?${params}` : ""}`;
+        console.log(url) //for debugging
+        const options = {
+            method: "get",
+            headers:{
+                "Accept": "application/json",
+                "apiKey": "your APIkey"
+            }
+        };
+        const response = await fetch(url, options);
+        console.log(`${response.status} (${response.statusText})`) // for debugging
+        const json = await response.json();
+        return json;
+    }
+}
+
+module.exports = Graph;
+
+```
+Note that we hard-coded the SAP Graph server that uses SAP API Business Hub sandbox. This allows us to focus here on the data access API, without requiring you to first configure your own SAP Graph business data graph and deal with all the complex aspects of security and authentication, which will be the subject of future tutorial parts.
```

Comment:
Use You and not the royal We.(unless, in rare cases, you are referring to SAP). Same thing with "us".

Reply:
Rephrased.